scylla-cql/src/frame/frame_errors.rs

```diff
@@ -109,6 +111,18 @@ pub enum SchemaChangeEventParseError {
     UnknownTargetOfSchemaChange(String),
 }
 
+/// An error type returned when deserialization of [Status/Topology]ChangeEvent fails.
+#[non_exhaustive]
+#[derive(Error, Debug)]
+pub enum ClusterChangeEventParseError {
+    #[error("Malformed type of status change: {0}")]
+    TypeOfChangeParseError(LowLevelDeserializationError),
+    #[error("Malformed node address: {0}")]
+    NodeAddressParseError(LowLevelDeserializationError),
+    #[error("Unknown type of status change: {0}")]
+    UnknownTypeOfChange(String),
+}
```

Comment:
You mention both `Topology` and `Status` in the docstring but only mention `status` in error messages. Why so?

Reply:
At least `TypeOfChangeParseError` should have `cluster change` instead of `status change`.